<launch>

    <node pkg="final" type="cv_node.py" name="CVNode" output="screen">
        <param name="sub_topic" type="string" value="/camera/color/image_raw" />
        <param name="pub_topic_red" type="string" value="/camera/color/image_filtered_red" />
        <param name="pub_topic_blue" type="string" value="/camera/color/image_filtered_blue" />
        <param name="Debug" type="bool" value="False" />
    </node>

</launch>